### DISCUSSION TITLE
[求助]安东丽娜青衣这套电队怎么样

### DISCUSSION THREAD

**Discussion by sz0628** [share121/inter-knot#2987] [其他]
b站上看到有人说安东和格莉斯不适合一队，所以把他们分开了，击破位放青衣可以吗

**Comment by break-under-star** [comment]
青衣击破通用，谁都能用<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
